runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8274,7 +8294,9 @@ done:;
 			}
 			_pc += 3;
 			*(j9object_t*)--_sp = instance;
-
+			if (J9_ARE_ALL_BITS_SET(resolvedClass->classFlags, J9ClassContainsUnflattenedFlattenables)) {
```

Comment:
also put VT ifdefs around this

Reply:
Does VT stand for VTABLE_SLOT_TAG_MASK ?

Reply:
vt -> valuetypes, `#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)`